docs/docs/server/conversations.md

```diff
@@ -11,6 +11,7 @@ The server has the following endpoints:
   conversation before you can send messages to the chatbot.
 - `POST /conversations/:conversationId/messages` sends a message to the chatbot,
   and gets a response back.
+- `GET /conversations/:conversationId` gets a conversation in its current state.
```

Comment:
Do we want to note the security implications here or elsewhere?

i.e. in a naïve implementation, if a malicious actor got a hold of your conversation id they could read your entire conversation history so you should probably add some middleware to enforce authn/authz.

Reply:
good point. will add documentation